env_jax.yml

```diff
@@ -0,0 +1,10 @@
+name: deepchem
+channels:
+  - omnia
+  - conda-forge
+  - defaults
+dependencies:
+  - numpy==1.19.*
```

Comment:
Does the latest jax run on numpy 1.20 or 1.19? You can test this by installing jax in a fresh environment and seeing which numpy it pulls down

Reply:
Yup, it works well.

Reply:
Do you mean the latest jax uses numpy 1.19?

Reply:
I'm sorry if I might have misunderstood the question, but from its `setup.py` file it doesn't mention any specific depedency on numpy 1.20 or 1.19 - https://github.com/google/jax/blob/6382cee7084ebea2628ca139cbaf7415108cd78f/setup.py#L37

I even ran a simple collab notebook and it works perfectly fine in `numpy 1.19`  with latest version of `jax 2.14`- [google collab](https://colab.research.google.com/drive/1Iwd2B8wTqLBQSS4t4OWvmh14p5jSCAD_?usp=sharing)